Source/WebKit/Shared/WebEvent.h

```diff
@@ -44,7 +44,7 @@ namespace WebKit {
 class WebEvent {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    enum Type {
+    enum class Type {
```

Comment:
While we're here, let's make this a `uint8_t` as well to match `WebCore::PlatformEvent::Type`.
```
-    enum Type {
+    enum class Type : uint8_t {
```